### ISSUE TITLE
Emergency care card has low contrast when printing

### ISSUE THREAD

**Issue by anandamaryon1** [nhsuk/nhsuk-service-manual-community-backlog#533]
## What is the problem?

The emergency care card, from the "Help users decide when and where to get care (care cards)" pattern (https://github.com/nhsuk/nhsuk-service-manual-community-backlog/issues/8) lacks contrast when being printed out. 

Tested via print to PDF only. Will need to test with a real printer and different devices when fixing this bug.

## Attachments

nhs.uk example
![image](https://github.com/user-attachments/assets/550e04fa-9113-4782-959e-62979b62dd31)

nhsuk-frontend / design system example
![image](https://github.com/user-attachments/assets/7df767de-5ace-44ba-8c95-b8a0d836fa69)

With background images turned on
![image](https://github.com/user-attachments/assets/255f3c71-29fd-4e4f-a844-24d3821e85b7)

**Comment by anandamaryon1** [comment]
Fixed in frontend release 9.0.1: https://github.com/nhsuk/nhsuk-frontend/releases/tag/v9.0.1
